src/main/java/org/testng/internal/XmlMethodSelector.java

```diff
@@ -231,11 +187,19 @@ private boolean hasExcludedMethods() {
     return vResult;
   }
 
-  private String makeMethodName(String className, String methodName) {
+  private static String methodName(String methodName) {
```

Comment:
Yes, it is not an obvious part. In fact, I need it if I want to pass all test cases I made. I accept help and idea to improve it :)